### DISCUSSION TITLE
Req: Limit R (Lora Down output) per channel by weight's absolute value

### DISCUSSION THREAD

**Discussion by torridgristle** [hako-mikan/sd-webui-lora-block-weight#178] [Ideas]
Limiting the blocks used from a Lora has made many Loras much more useful.

There may be another way to further limit what a Lora does by taking the absolute value of the Lora Down weight (input features to R weight) and zeroing anything above or below a certain value.

Ideally this would allow the user to disable any small adjustments and retain only the most prominent elements, or disable any large adjustments that may be unwanted in some blocks and keep the rest.

I don't know if this would actually work well.

**Comment by hako-mikan** [comment]
Well, this is an interesting method. 
Thanks for the suggestion.
